COVIDWatch iOS/ViewControllers/Bluetooth.swift

```diff
@@ -24,23 +24,12 @@ class Bluetooth: BaseViewController {
         img.center.y = header.frame.minY + (282.0/812.0) * screenHeight
         view.addSubview(img)
 
-        largeText.text = "Quickly Connect"
-        largeText.frame.size.height = largeText.contentSize.height
-        largeText.frame.origin.y = header.frame.minY + (481.0/812.0) * screenHeight
-        largeText.center.x = view.center.x
-        view.addSubview(largeText)
+        largeText.draw(parentVC: self, centerX: view.center.x, centerY: header.frame.minY + (512.0/812.0) * screenHeight)
```

Comment:
Do you think it would be easier to maintain if we moved these numbers to variables so they have a name?  It will probably be hard to know what 512 to 812 represents in 6 months.  :)

Reply:
Yes it would